### PR TITLE
new playfield, build after interface specified in issue about playfield

### PR DESCRIPTION
**Pull request by MarieTheresThomas**
contains the new file "updatedGame.json" in client>>src>>assets, which specifies all following JSON format: 
        {
            "name" : "Mediterranean Avenue",
            "type" : "street",
            "group" : [1, 1, 2],    (means:  Group "1", member "1", total of "2" members in Group)
            "price" : 60,
            "house_price" : 50,
            "rent" : [2,10,30,90,160,250],   (means: rent for 1 house, 2 houses, 3...)
            "id":2
        },
Names  are just placeholders (directly copied from original monopoly), Fields 1-40 are in correct order,

### CODE REVIEW

File: client/src/assets/updatedGame.json [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+        {
+            "name" : "start",
+            "type" : "start", 
+            "group" : 0,
+            "id": 1
+        },
+        {
+            "name" : "Mediterranean Avenue",
+            "type" : "street",
+            "group" : [1, 1, 2],
```
Comment (ratnim) [inline client/src/assets/updatedGame.json]: #90 defines group as an integer.

File: client/src/assets/updatedGame.json [hunk 2]
```diff
@@ -0,0 +1,308 @@
+{
+    "fields":[
+        {
+            "name" : "start",
+            "type" : "start", 
+            "group" : 0,
+            "id": 1
```
Comment (ratnim) [inline client/src/assets/updatedGame.json]: #90 Fields are in sorted. Therefore no id is needed.

File: client/src/assets/updatedGame.json [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+            "name" : "Income Tax",
+            "type" : "tax",
+            "price" : 200,
+            "id":5
+        },
+        {
+            "name" : "Reading Railroad",
+            "type" : "station",
+            "group" : [9, 1, 4],
+            "price" : 200,
```
Comment (ratnim) [inline client/src/assets/updatedGame.json]: Stations have rents too.

File: client/src/assets/updatedGame.json [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+            "group" : [1, 2, 2],
+            "price" : 60,
+            "house_price" : 50,
+            "rent" : [4,20,60,180,320,450],
+            "id":4
+        },
+        {
+            "name" : "Income Tax",
+            "type" : "tax",
+            "price" : 200,
```
Comment (ratnim) [inline client/src/assets/updatedGame.json]: Income tax should have a rent but no price.

File: client/src/assets/updatedGame.json [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+            "name" : "Oriental Avenue",
+            "type" : "street", 
+            "group" : [2, 1, 3],
+            "price" : 100,
+            "house_price" : 50,
+            "rent" : [6,30,90,270,400,550],
+            "id":7
+        },
+        {
+            "name" : "chance",
```
Comment (ratnim) [inline client/src/assets/updatedGame.json]: Should start with a capital letter. "Chance"

File: client/src/assets/updatedGame.json [hunk 6]
```diff
@@ -0,0 +1,308 @@
...
+            "price" : 140,
+            "house_price" : 100,
+            "rent" : [10,50,150,450,625,750],
+            "id":12
+        },
+        {
+            "name" : "Electric Company",
+            "type" : "utility",
+            "group" : [10, 1, 2],
+            "price" : 150,
```
Comment (ratnim) [inline client/src/assets/updatedGame.json]: Rent should be defined.

File: server/assets/boards/berlin.json [hunk 7]
```diff
@@ -0,0 +1,273 @@
...
+            "name" : "Income Tax",
+            "type" : "tax",
+            "rent" : [200]
+        },
+        {
+            "name" : "Westhafen",
+            "type" : "station",
+            "group" : 8,
+            "price" : 200,
+            "rent" : [50]
```
Comment (Highlife1911) [inline server/assets/boards/berlin.json]: rent should be 50, 100, 150, 200

File: server/assets/boards/berlin.json [hunk 8]
```diff
@@ -0,0 +1,273 @@
...
+        {
+            "name" : "Wedding",
+            "type" : "street", 
+            "group" : 2,
+            "price" : 140,
+            "house_price" : 100,
+            "rent" : [10,50,150,450,625,750]
+        },
+        {
+            "name" : "Electric Company",
```
Comment (Highlife1911) [inline server/assets/boards/berlin.json]: Brandenburger Tor

File: server/assets/boards/berlin.json [hunk 9]
```diff
@@ -0,0 +1,273 @@
...
+        {
+            "name" : "Bergmannkiez",
+            "type" : "street",
+            "group" : 5,
+            "price" : 260,
+            "house_price" : 150,
+            "rent" : [22,110,330,800,975,1150]
+        },
+        {
+            "name" : "Water Works",
```
Comment (Highlife1911) [inline server/assets/boards/berlin.json]: Fernsehturm

File: server/assets/boards/berlin.json [hunk 10]
```diff
@@ -0,0 +1,273 @@
...
+            "name" : "Mitte",
+            "type" : "street",
+            "group" : 7,
+            "price" : 400,
+            "house_price" : 200,
+            "rent" : [50,200,600,1400,1700,2000]
+        }
+
+    ]
+}
```
Comment (Highlife1911) [inline server/assets/boards/berlin.json]: newline

File: server/assets/boards/berlin.json (hunk 7 above)
Comment (ratnim) [reply server/assets/boards/berlin.json]: Imo it should be the base rent. And the client and server know the rest to calculate the rent for the station.

**Pull request merged**